daras_ai_v2/language_model.py

```
@@ -2490,28 +2496,41 @@ def entry_to_prompt_str(entry: ConversationEntry) -> str:
+    content = []
     if input_images:
-        content = [
-            {"type": "image_url", "image_url": {"url": url}} for url in input_images
-        ] + [
-            {"type": "text", "text": content},
-        ]
+        content.extend(
+            [{"type": "image_url", "image_url": {"url": url}} for url in input_images]
+        )
+    if input_audio:
+        r = requests.get(input_audio)
```

Ruff (0.14.5)

Probable use of `requests` call without timeout

(S113)